static/app/views/metrics/layout.tsx

```diff
@@ -164,25 +113,23 @@ export const MetricsLayout = memo(() => {
             <OnboardingPanel image={<EmptyStateImage src={emptyStateImg} />}>
               <h3>{t('Track and solve what matters')}</h3>
               <p>
-                {t(
-                  'Create custom metrics to track and visualize the data points you care about over time, like processing time, checkout conversion rate, or user signups. See correlated trace exemplars and metrics if used together with Performance Monitoring.'
+                {tct(
+                  'Query and plot metrics extracted from your span data, to visualise trends and identify anomalies. To get started, you need to enable [link:tracing].',
```

Comment:
```suggestion
                  'Query and plot metrics extracted from your span data to visualise trends and identify anomalies. To get started, you need to enable [link:tracing].',
```
I figured out we probably don't need that comma.